contrib/fwupd-efi.spec.in

```diff
@@ -0,0 +1,70 @@
+%define alphatag                #ALPHATAG#
+
+%global enable_ci 0
+%global debug_package %{nil}
+
+Summary:   Firmware update daemon EFI binaries
+Name:      fwupd-efi
+Version:   #VERSION#
+Release:   0.#BUILD#%{?alphatag}%{?dist}
+License:   LGPLv2+
+URL:       https://github.com/fwupd/fwupd-efi
+Source0:   http://people.freedesktop.org/~hughsient/releases/%{name}-%{version}.tar.xz
+
+# these are the only architectures supporting UEFI SecureBoot
+ExclusiveArch: x86_64 aarch64
+
+#BuildRequires: elfutils-libelf-devel
+BuildRequires: meson
+BuildRequires: gnu-efi-devel
+BuildRequires: pesign
+
+Requires: fwupd
```

Comment:
Should it be the other way around?  `fwupd` requires `fwupd-efi`?

Reply:
I wondered about `Recommends`, but you're right, it's the other way around.